resource/js/vocab-search.js

```diff
@@ -6,49 +6,246 @@ const vocabSearch = Vue.createApp({
       languages: [],
       selectedLanguage: null,
       searchTerm: null,
-      autoCompeteResults: [],
-      languageStrings: null
+      searchCounter: null,
+      renderedResultsList: [],
+      languageStrings: null,
+      msgs: null
     }
   },
   mounted () {
     this.languages = window.SKOSMOS.languageOrder
     this.selectedLanguage = window.SKOSMOS.content_lang
-    this.languageStrings = window.SKOSMOS.language_strings[window.SKOSMOS.lang]
+    this.searchCounter = 0
+    this.languageStrings = window.SKOSMOS.language_strings[window.SKOSMOS.lang] ?? window.SKOSMOS.language_strings.en
+    this.msgs = window.SKOSMOS.msgs[window.SKOSMOS.lang] ?? window.SKOSMOS.msgs.en
+    this.renderedResultsList = []
+    document.addEventListener('click', this.onClickOutside)
```

Comment:
This (and line 25) could be done like in for example hierarchy tab using a custom directive: https://github.com/NatLibFi/Skosmos/blob/3925c7162660286cb20903e73c129dbed4a1669f/resource/js/tab-hierarchy.js#L170-L180

It is probably not necessary and could be left as it is, though